doc/media.md

```diff
@@ -252,7 +259,7 @@ startMedia(mediaReqOptions?: IMediaRequestOptions, retryOnFailure?: boolean): Pr
 Params:
 * `mediaReqOptions?: IMediaRequestOptions` – Optional: defaults to `{ video: true, audio: true }` –
   request video and/or audio with a default device or deviceId. See [IMediaRequestOptions] for more details.
-* `retryOnFailure?: boolean` – Optional: default `true` – whether the sdk should retry on an error
+* `retryOnFailure?: boolean` – Optional: (this option is deprectated – use `mediaReqOptions.retryOnFailure`) default `true` – whether the sdk should retry on an error
```

Comment:
This is going to be a major version release anyway right? If that's the case, I say we just move this and *not* deprecated it.

Reply:
I wasn't actually going to make this a major version release which is why I deprecated it. I was going to try and release this and the major release was going to be the persistent connection work.